organization/_posts/2021/2021-03-04-coordination.md

```diff
@@ -0,0 +1,160 @@
+
+*DOMA* - Rucio, File Transfer Service (FTS), Storage interfaces and caching layers (CTA, dCache, Xcache, StoRM, xrootd, http TPC, etc.), Network technologies including monitoring and software defined networks, CVMFS, Token based authentication. Liz and Graeme made a presentation at the last DOMA general meeting (<https://indico.cern.ch/event/1009737/>). We will assemble an editorial team.
+
+Feedback/discussion from LHCC and review chair:
+
+- Generally they are happy with the plan
+  - "The LHCC thanks the WLCG PI, WLCG SW liaisons, the experiments and the HSF for their efforts to come up with a sensible list of Common Software Activities. We understand the schedule is tight and we appreciate your help to make this review a success."
+  - Amber praised the clear selection of projects and the fact that our documents will be useful beyond the review
+- We did discuss some specific projects/issues
+  - Accelerators? Yes, but on a per-project basis, rather than generally
+  - Build systems? Probably not, as the scaling for HL-LHC is not a big problem (albeit, likely to need optimised builds)
+    - Meeting discussed this and we shall raise this again with the WLCG/Experiments; could foresee to have it in the foundation document
+  - ACTS? To be reviewed later, in the ATLAS context as other LHC experiments not using it
+    - Noted that as this is an LHCC review the non-LHC use of ACTS, while very welcome, is not in scope
+
+### HSF Plans
+
+As mentioned above... the summary of our plans for 2021 are here: <https://hepsoftwarefoundation.org/organization/planning/plan-of-work-2021.html>.
+
+Please help circulate it to other communities.
+
+### Google Summer of Code, Season of Docs
+
+- Applied as CERN-HSF Organization, accepted orgs announced Mar 9
+  - 37 project proposals (vs. 47 last year)
+  - Students will contact mentors from **Mar 9** (>400 registered in the Gitter chat channel!)
+  - First phase evaluation mandatory based on a test, **Mar 25** is the limit date for mentors to announce the results.
+  - Extensive interaction mentor-student for the students passing the test
+  - *Clarification:* not necessary to put the test on the webpage, it can be communicated directly to students
+- Season of Docs 2021 call for mentors and project proposals already issued
+  - Proposals accepted until **Mar 25** (on Mar 26 we need to apply as Org)
+  - We had 4 projects in 2020 (2 short-term, 2 long-running) on [Rucio](https://www.divya-mohan.com/project-report-gsod), [Allpix Squared](https://github.com/sabitarao/gsod/wiki), ROOT (being evaluated) 
+    - Techical writer [blogs](https://hepsoftwarefoundation.org/gsdocs/2020/blogs.html) available
+  - The web page 2021 still being prepared. Folders in the repo for 2021 not there yet, but people can [contact us](mailto:hsf-gsdocs-admin@googlegroups.com) in case they plan to make a proposal.
```

Comment:
```suggestion
  - The web page is 2021 still being prepared. Folders in the repo for 2021 are not there yet, but people can [contact us](mailto:hsf-gsdocs-admin@googlegroups.com) in case they plan to make a proposal.
```